ios/RNCAsyncStorage.m

```diff
@@ -320,7 +341,10 @@ - (instancetype)init
   
   // Then migrate what's in "Documents/.../RCTAsyncLocalStorage_V1" to "Application Support/[bundleID]/RCTAsyncLocalStorage_V1"
   RCTStorageDirectoryMigrationCheck(RCTCreateStorageDirectoryPath_deprecated(RCTStorageDirectory), RCTCreateStorageDirectoryPath(RCTStorageDirectory), NO);
-
+    
+  // exclude from backup
+  //setBackupEnabled(RCTCreateStorageDirectoryPath(RCTStorageDirectory), false);
```

Comment:
If you want to test this feature, uncomment this line first as well.

Reply:
Will this be settable from JS? How do we surface it to consumers?

Reply:
I'm yet to find best way to do so. Though maybe through setting this as `package.json` entry (and we could actually move all feature flags there). Got any suggestions?